mewarpx/mewarpx/diags_store/flux_diagnostic.py

```diff
@@ -610,7 +610,7 @@ def __init__(self, diag_steps, runinfo,
                 period.
             profile_decorator (decorator): A decorator used to profile the
                 timeseries update methods and related functions.
-            kwargs: See :class:`metools.diags_store.diag_base.WarpDiagnostic`
+            kwargs: See :class:`mewarpx.diags_store.diag_base.WarpXDiagnostic`
```

Comment:
Looks like there are still a few metools strings in docs so a good opportunity to do a find/replace